src/app/user-management/update-user/update-user.component.html

```diff
@@ -2,12 +2,13 @@
   <div class="modal-background"></div>
   <div class="modal-card">
     <header class="modal-card-head">
-      <p class="modal-card-title">Update User</p>
+      <p *ngIf="showRoleSection" class="modal-card-title">Update user role</p>
+      <p *ngIf="!showRoleSection" class="modal-card-title">Update user password</p>
```

Comment:
what is `Update user password` ? toggling?

Reply:
toggling of card title based on button action  `reset password` or `change role`